README.md

```diff
@@ -21,6 +21,55 @@ A flexible and scalable platform for running Kubernetes control plane APIs.
 - Single binary CLI for improved user experience:
     - initialize, install operator, manage lifecycle of control planes and contexts.
 
+
+## Prerequisites
+Before you begin, ensure you have the following tools installed and configured:
+
+- **[kind](https://kind.sigs.k8s.io)**: Required for the default hosting cluster setup.
+- **[kubectl](https://kubernetes.io/docs/tasks/tools/)**: Required for interacting with Kubernetes clusters.
```

Comment:
The URL here could point specifically to the section about kubectl: https://kubernetes.io/docs/tasks/tools/#kubectl